hera_mc/geo_handling.py

```diff
@@ -144,38 +144,44 @@ def is_in_geo_location(self, station_name):
             station_present = False
         return station_present
 
-    def is_in_connections(self, station_name, active_date=None):
+    def is_in_connections(self, station_name, active_date=None, return_antrev=True):
```

Comment:
I sort of agree.  I did modify it from the trickiness of return boolean if active_date was none otherwise return value (that was not a good thing...).  I have been cleaning up geo_handling and cm_handling (I've done a bit already on geo_handling), so I'd like to defer this to the future as I continue cleaning...